Build failure in CI: LLVM AOT cross-compile CoreCLR tests

Failed in PRs: #66552, #66528, #66519 .. and more.

logs: https://dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_apis/build/builds/1660954/logs/1459

```
2022-03-13T13:21:39.0684383Z   /__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/opt: mono_aot_IY6j2Q/temp.bc: error: Invalid record (Producer: 'LLVM11.1.0' Reader: 'LLVM 11.1.0')
2022-03-13T13:21:39.0701714Z   AOT of image /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/Performance/CodeQuality/Burgers/Burgers/Burgers.dll failed.
2022-03-13T13:21:39.0704770Z   Mono Ahead of Time compiler - compiling assembly /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/Performance/CodeQuality/Burgers/Burgers/Burgers.dll
2022-03-13T13:21:39.0706214Z   AOTID 2DFE6CBE-B22D-72B1-15E8-31EC018625D8
...
2022-03-13T13:24:58.5100636Z   /__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/opt: mono_aot_J3nrZB/temp.bc: error: Invalid record (Producer: 'LLVM11.1.0' Reader: 'LLVM 11.1.0')
2022-03-13T13:24:58.5138159Z   AOT of image /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_r/CreateGeneric_r.dll failed.
2022-03-13T13:24:58.5162401Z   Mono Ahead of Time compiler - compiling assembly /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_r/CreateGeneric_r.dll
2022-03-13T13:24:58.5164465Z   AOTID A6DB805E-EA5E-84B0-4061-6AFD48D765F3
2022-03-13T13:24:58.5166773Z   Executing opt: "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/opt" -f -O2 -disable-tail-calls -place-safepoints -spp-all-backedges -mattr=crc,crypto -o "mono_aot_J3nrZB/temp.opt.bc" "mono_aot_J3nrZB/temp.bc"
2022-03-13T13:24:58.5201304Z /__w/1/s/src/mono/msbuild/aot-compile.proj(19,9): error MSB3073: The command "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/mono-aot-cross /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_r/CreateGeneric_r.dll" exited with code 1.
2022-03-13T13:24:58.5308719Z   aot-compile: compiling /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_ro/CreateGeneric_ro.dll; MONO_PATH: /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_ro:/__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/Tests/Core_Root
2022-03-13T13:24:58.8823507Z   /__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/opt: mono_aot_RKtyf6/temp.bc: error: Invalid record (Producer: 'LLVM11.1.0' Reader: 'LLVM 11.1.0')
2022-03-13T13:24:58.8854385Z   AOT of image /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_ro/CreateGeneric_ro.dll failed.
2022-03-13T13:24:58.8857069Z   Mono Ahead of Time compiler - compiling assembly /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_ro/CreateGeneric_ro.dll
...
2022-03-13T13:57:12.8597498Z Build FAILED.
2022-03-13T13:57:12.8598100Z 
2022-03-13T13:57:12.8601671Z /__w/1/s/src/mono/msbuild/aot-compile.proj(19,9): error MSB3073: The command "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/mono-aot-cross /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/Performance/CodeQuality/Burgers/Burgers/Burgers.dll" exited with code 1.
2022-03-13T13:57:12.8605874Z /__w/1/s/src/mono/msbuild/aot-compile.proj(19,9): error MSB3073: The command "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/mono-aot-cross /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_r/CreateGeneric_r.dll" exited with code 1.
2022-03-13T13:57:12.8609903Z /__w/1/s/src/mono/msbuild/aot-compile.proj(19,9): error MSB3073: The command "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/mono-aot-cross /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/CreateGeneric_ro/CreateGeneric_ro.dll" exited with code 1.
2022-03-13T13:57:12.8614258Z /__w/1/s/src/mono/msbuild/aot-compile.proj(19,9): error MSB3073: The command "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/mono-aot-cross /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/VectorUnused_r/VectorUnused_r.dll" exited with code 1.
2022-03-13T13:57:12.8617941Z /__w/1/s/src/mono/msbuild/aot-compile.proj(19,9): error MSB3073: The command "/__w/1/s/artifacts/bin/mono/Linux.arm64.Release/cross/linux-arm64/mono-aot-cross /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Release/JIT/SIMD/VectorUnused_ro/VectorUnused_ro.dll" exited with code 1.
2022-03-13T13:57:12.8619671Z     0 Warning(s)
2022-03-13T13:57:12.8620314Z     5 Error(s)
```

--- a/src/mono/mono/mini/simd-intrinsics.c
+++ b/src/mono/mono/mini/simd-intrinsics.c
@@ -3694,10 +3694,6 @@ mono_emit_simd_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 	if (!strcmp (class_ns, "System.Numerics") && !strcmp (class_name, "Vector")){
 		return emit_sri_vector (cfg, cmethod, fsig, args);
 	}
-
-	if (!strcmp (class_ns, "System.Numerics") && !strcmp (class_name, "Vector`1")){
-		return emit_vector64_vector128_t (cfg, cmethod, fsig, args);
-	}
 #endif // defined(TARGET_ARM64)
 
 	return emit_simd_intrinsics (class_ns, class_name, cfg, cmethod, fsig, args);

